executor2.py

```diff
@@ -143,6 +143,19 @@ def logger(self):
             'buildJobId': self._config['BUILD_JOB_ID'],
             'buildJobConsoles': []
         }
+
+        # Add a hidden version notice.
```

Comment:
A hidden notice so its easier to identify the executor. This will show up in logs.